modules/network-observability-total-resource-usage.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+// * network_observability/configuring_operator.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-total-resource-usage-table_{context}"]
+= Total average memory and CPU usage
+
+The following table outlines averages of total resource usage for clusters with a sampling value of 1, 50, and 400 for 3 different tests: `Test 1`, `Test 2`, and `Test 3`. The tests differ in the following ways:
+
+- `Test 1` takes into account the total number of namespace, pods and services in {product-title} cluster, places load on the eBPF agent, and represents use cases with a high number of workloads for a given cluster size. For example, `Test 1` consists of 76 Namespaces, 5153 Pods, and 2305 Services.
+- `Test 2` takes into account a high ingress traffic throughput.
```

Comment:
Could "throughput" be replaced with "capacity" ? Not sure, but tbh had to look up definition of that word. :)

Reply:
then maybe "bandwidth" rather than "capacity", would that work for you?

Reply:
Hello @jotak,
   Thanks for the response. I think either would work...and to be honest, "throughput" was not a term that I was familiar with...maybe others would be, just trying to make it as clear as possible. Thanks!

Reply:
Switched it up with "volume." Thanks for the suggestion. :)